### ISSUE TITLE
Is there any way we can get the vocabulary-topic distribution matrix?

### ISSUE THREAD

**Issue by sharon-gao** [maria-antoniak/little-mallet-wrapper#5]
Hi Maria,

Thanks for your development, that is really helpful for people used to coding in Python! 

I wonder whether there could be an output of the vocabulary scores for each topic, instead of only the top twenty words to represent the topics. Sometimes it would be useful to know the scores in order to evaluate the model quality.

Best

**Comment by maria-antoniak** [comment]
Yes, I've also realized that we need this! Adding it to the to-do list and it should be included in the next version.
